meshroom/ui/qml/GraphEditor/NodeEditor.qml

```diff
@@ -17,30 +17,42 @@ Panel {
     property variant node
     property bool readOnly: false
     property bool isCompatibilityNode: node && node.compatibilityIssue !== undefined
+    property string nodeStartDateTime : ""
 
     signal attributeDoubleClicked(var mouse, var attribute)
     signal upgradeRequest()
 
     title: "Node" + (node !== null ? " - <b>" + node.label + "</b>" : "")
     icon: MaterialLabel { text: MaterialIcons.tune }
 
+    onNodeChanged: {
+        nodeStartDateTime=""
+        if(node !== null && node.isSubmittedOrRunning()){
+            timer.start()
+        }
+        else if (node !== null && node.isFinishedOrRunning()) {
+            computationInfo.text = Format.getTimeStr(node.elapsedTime)
```

Comment:
Seems to be the right thing to do here.
But I'm not sure if elapsedTime is correctly updated when the node is running for now.